src/Microsoft.DotNet.Darc/src/DarcLib/AzureDevOpsClient.cs

```diff
@@ -768,6 +768,16 @@ private VssConnection CreateVssConnection(string accountName)
         /// </remarks>
         public static (string accountName, string projectName, string repoName) ParseRepoUri(string repoUri)
         {
+            // If repoUri includes the user in the account we remove it otherwise Regex matching would fail for URIs like
```

Comment:
Can you add an issue to remove this later, with a link to the issue in comment here?

Reply:
Done